### PR TITLE
PAYARA-2991 Document Create a MicroProfile Settings Page into the Admin Console

### PR DESCRIPTION
**Pull request by MeroRai**

### CODE REVIEW

File: documentation/microprofile/healthcheck.adoc [hunk 1]
```diff
@@ -23,11 +23,23 @@ This endpoint can be altered using the `set-microprofile-healthcheck-configurati
...
+
+_Since *5.183*&nbsp;_
+
+[[using-the-admin-console]]
+=== Using the Admin Console
+
+To configure the Health Check in the Admin Console, go to Configuration 
+→ [instance-configuration (like server-config)] → MicroProfile → Health Check:
+
+image:/images/microprofile/fault-tolerance.png[Set Fault Tolerance Configuration]
```
Comment (OndrejM) [inline documentation/microprofile/healthcheck.adoc]: This should link to health check image

**Comment by OndrejM** [comment]
I'll fix after merging

File: documentation/microprofile/healthcheck.adoc (hunk 1 above)
Comment (MeroRai) [reply documentation/microprofile/healthcheck.adoc]: @OndrejM Made a PR to fix this https://github.com/payara/Payara-Server-Documentation/pull/415

Comment (OndrejM) [reply documentation/microprofile/healthcheck.adoc]: Thanks, @MeroRai

**Pull request merged**